src/common/regions/utils.ts

```diff
@@ -132,8 +142,9 @@ export function filterUndefinedRegions(regions: any[]): Region[] | [] {
 export function getAutocompleteRegionsWithGeolocation(
   geolocation: GeolocationInfo,
   locations: Region[],
+  countyToZipMap: CountyToZipMap,
```

Comment:
it doesnt look like youre passing `countyToZipMap` in when `getAutocompleteRegionsWithGeolocation` is called, and so geolocated regions arent populating in the search bar dropdown menu (they're meant to be at the top of the dropdown, see prod)

Reply:
thanks for pointing out, fixed, I think.  I can't test this locally because of CORS restrictions, and it looks to be the same on vercel (is there any way around this?).

Unfortunately I have a standing conflict with standup and can't attend, perhaps we could discuss via slack, or on this PR?

Reply:
I don't actually think you should have to reference the public data url [here](https://github.com/covid-projections/covid-projections/pull/2673/files#diff-d34ed76dc27ab34157f38096e85edb2ca75c0dbd8fb462dae9b250b2394b9070R4).  Pretty sure just `/data/county-zipcode.json` should work and it will server it locally correctly?

Reply:
@schamp ah yeah, geolocation doesnt work locally or in preview links. if you go into `useGeolocation.tsx`, there's a comment at the top explaining how to test locally -- uncomment `mockGeolocationData` (line 62), and in the effect on line 31 replace `fetchGeolocationData()` with `mockGeolocationData()`

Reply:
ah different CORS issue 😬

Reply:
@ghop02 good call on the public data url, that works as advertised.  
@chasulin thanks for the guidance, with that change, I've confirmed in the debugger the zip loading is working with the geolocation 👍